pkg/operator/observe_config_test.go

```diff
@@ -0,0 +1,44 @@
+package operator
+
+import (
+	"fmt"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/rest"
+)
+
+func TestObserveClusterConfig(t *testing.T) {
+
```

Comment:
and while you're updating, you need to kill your whitespace bunny that keeps putting these here.